### PR TITLE
Feature/logflare

### PR DESCRIPTION
**Pull request by DigneshGujarathi**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,3 +2,5 @@ REACT_APP_API_PREFIX="api"
 REACT_APP_GLIFIC_API_PORT=4000
 REACT_APP_APPSIGNAL_API_KEY=""
 REACT_APP_APPLICATION_NAME="Glific: Two way communication platform"
+LOGFLARE_API_KEY = 'vV5sj8-sZZIa'
```
Comment (mdshamoon) [inline .env.example:5]: Please remove from API keys from here

File: src/config/logs.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import pino from 'pino';
+import { createPinoBrowserSend, createWriteStream } from 'pino-logflare';
+
+const apiKey: string = 'vV5sj8-sZZIa';
```
Comment (mdshamoon) [inline src/config/logs.ts:4]: Remove from here too

File: .env.example (hunk 1 above)
Comment (kurund) [reply .env.example]: @DigneshGujarathi Can you also please change the API key in logflare.

File: src/containers/Auth/ConfirmOTP/ConfirmOTP.tsx [hunk 3]
```diff
@@ -73,8 +74,21 @@ export const ConfirmOTP: React.SFC<ConfirmOTPProps> = (props) => {
...
-      .catch(() => {
+      .catch((error) => {
         setAuthError('We are unable to register, kindly contact your technical team.');
+        // add log's
+        setLogs(
+          `onSubmitOTP:${{
+            user: {
+              name: props.location.state.name,
+              phone: props.location.state.phoneNumber,
+              password: props.location.state.password,
```
Comment (mdshamoon) [inline src/containers/Auth/ConfirmOTP/ConfirmOTP.tsx:85]: Should we have password in logs?

**Comment by mdshamoon** [comment]
Looks good.

**Pull request merged**